### DISCUSSION TITLE
Plataforma para desplegar mi proyecto

### DISCUSSION THREAD

**Discussion by DaveOval** [Launch-X-Latam/MisionFrontEnd#227] [Q&A] [answered]
Hola colegas, les quiero preguntar si conocen una plataforma para poder desplegar mi pagina que estoy creando para la practica de la pastelería, ya tengo un avance y me gustaría mostrarlo pero no tengo una idea de como lo podría hacer. De preferencia que sea gratuito.
Muchas gracias de antemano y un saludo cordial.

**Comment by doguedogue** [comment, marked as answer]
Hola mira hay muchisimas, busca en youtube como utilizarlas, realmente es muy intuitivo: 
- GitHub pages (aqui mismo en GitHub en tu repositorio donde colocas tu código)
- Heroku
- Netlify
- Azure
- etc.

🚀🚀🚀

**Reply by JoseEduardoVelazquezCN** [reply]
Realmente netlify es muy sencillo incluso si ya tienes tu página en un tu repositorio pues ya solo es cuestión de unos cuantos clics o arrastrar tus archivos. 
Bueno es el que uso
